packages/core/strapi/lib/core/loaders/apis.js

```diff
@@ -115,6 +116,15 @@ const loadContentTypes = async (dir) => {
     const contentTypeName = normalizeName(fd.name);
     const contentType = await loadDir(join(dir, fd.name));
 
+    if (
+      isEmpty(contentType) ||
+      (isEmpty(contentType.schema) &&
+        isEmpty(contentType.actions) &&
```

Comment:
Are actions & lifecycles required though?
Shouldn't we only check if the schema is missing?

Reply:
In theory yeah, but I went with the least-strict option in case there was some weird edge case that would make this a breaking change

Reply:
But TBH if actions load but schema doesn't exist, the user would still see the "singularName" error message. So I might just make this strict as well.

I would have put the check in `validateContentTypesUnicity` where it belongs, but then it doesn't have access to `dir` or anything that could point the user to the actual problem, just an empty object 😔